src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/product.html.twig

```diff
@@ -160,7 +162,7 @@
         </div>
       {% endif %}
       {% for customization in product.customizations.textCustomizations %}
-        <p><strong>{{ customization.name }}</strong> {{ customization.value }}</p>
+        <p><strong>{{ customization.name }} :</strong> {{ customization.value }}</p>
```

Comment:
:warning: In english, there is no space before `:`

Reply:
Franck is right: English grammar requires no space between the last letter of a word and the following punctuation sign. ;-)